src/test/java/bio/terra/common/CloudPlatformWrapperTest.java

```diff
@@ -31,8 +31,14 @@ void choose_Map(CloudPlatform cloudPlatform, String expected) {
 
```

Comment:
not necessary but maybe we could fix the name of choose_Map -> chooseMap

Reply:
Good idea!